website/public/index.ejs.html

```diff
@@ -802,6 +802,128 @@ <h2 id="linter-feature-comparison">Features</h2>
           </p>
         </figcaption>
       </figure>
+
+      <figure class="comparison-mobile">
+        <table>
+          <thead>
+            <tr class="header">
+              <th class="quick-lint-js">quick-lint-js</th>
+              <th><a href="https://eslint.org/">ESLint</a></th>
+              <th><a href="https://jshint.com/">JSHint</a></th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td class="good quick-lint-js has-link">
+                <a href="benchmarks/"
+                  ><span aria-labelledby="feature-speed-qljs-quick">🚀</span
+                  ><br /><span id="feature-speed-qljs-quick">quick</span></a
```

Comment:
This link has underlines in the mobile version, which look ugly. The underlines don't appear in the desktop version. Remove the underlines.